### DISCUSSION TITLE
Local network

### DISCUSSION THREAD

**Discussion by Vyroum** [ElSnoMan/pyleniumio#315] [Q&A]
Hello, is there any way to use Pylenium in local network, like closed company nets? I dont have internet conection, but i really want to try Pylenium for automating testing

**Comment by ElSnoMan** [comment]
I'm not sure how you would do web automation without internet. If you can find how to do it with Selenium, then we can do it with Pylenium

**Comment by Vyroum** [comment]
I mean, i can manually download packages but, have restricted policy about ports. It seems some packages trying to download something on test startup, Maybe i can override webdriver manager (i.e) and use downloaded drivers directly?

**Comment by ElSnoMan** [comment]
In pylenium.json, you can set the local_path to your downloaded driver so
it doesn't check online to download it for you
